JPS_Ontology/ontology/ontoreaction/OntoReaction.csv

```diff
@@ -1,6 +1,6 @@
 Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
 OntoReaction,TBox,https://www.theworldavatar.com/kg/ontoreaction/,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
-OntoReaction,TBox,1.10,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoReaction,TBox,1.1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
```

Comment:
The version should be increased to `1.11`, not `1.1`.

Reply:
Noted. Will change accordingly.